roles/prometheus/templates/prometheus.yml.j2

```diff
@@ -0,0 +1,23 @@
+
+global:
+  scrape_interval: 15s 
+  evaluation_interval: 15s 
+
+# alerting:
+  # alertmanagers:
+  #   - static_configs:
+  #       - targets:
+          # - alertmanager:9093
+
+# rule_files:
+
+
+scrape_configs:
+
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:9090"]
+
+  - job_name: "node_exporter"
+    static_configs:
+      - targets: ['localhost:9100']
```

Comment:
Should this target url be parameterized and derived from the nodes where the node_exporter is installed?